<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RealtimeGraphMeasure1FormClass</class>
 <widget class="GUIFormBase" name="RealtimeGraphMeasure1FormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RealtimeGraphMeasure1Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>24</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>焦点レベル</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>381</width>
     <height>131</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <row>
    <property name="text">
     <string>Top</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Center</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Bottom</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Left</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Center</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Right</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>381</width>
     <height>24</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>輝度レベル</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetBrightness">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>381</width>
     <height>131</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <row>
    <property name="text">
     <string>Top</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Center</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Bottom</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Left</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Center</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Right</string>
    </property>
   </column>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GUIFormBase</class>
   <extends>QWidget</extends>
   <header>guiformbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
